client/state/selectors/get-editor-close-config.js

```diff
@@ -63,6 +63,13 @@ export default function getEditorCloseConfig( state, siteId, postType, fseParent
 		};
 	}
 
+	// If no postType, assume site editor and land on view.
+	if ( ! postType ) {
+		return {
+			url: `/view/${ getSiteSlug( state, siteId ) }`,
```

Comment:
Since in wp-admin we are returning to dashboard, I'd probably default this to `home` since I'm guessing that's going to be the previous page most of the time. The view looks a bit broken for me, and there won't be much value in it once we introduce the preview feature in site editor.

Reply:
That makes sense.  I have refactored to make this the fallback for the site editor as well.